doc/radosgw/adminops.rst

```diff
@@ -1354,6 +1358,21 @@ Request Parameters
 :Example: True [False]
 :Required: No
 
+``max-entries``
+
+:Description: A number of bucket list entries to return, max is
+              enforced to rgw_list_buckets_max_chunk config option.
```

Comment:
The number of bucket list entries to return, cannot be more than ``rgw_list_buckets_max_chunk``